src/mono/mono/eventpipe/ds-rt-mono.h

```diff
@@ -6,45 +6,52 @@
 
 #ifdef ENABLE_PERFTRACING
 #include "ep-rt-mono.h"
+#include <mono/utils/mono-logger-internals.h>
 
 #undef DS_LOG_ALWAYS_0
-#define DS_LOG_ALWAYS_0(msg)
+#define DS_LOG_ALWAYS_0(msg) mono_trace(G_LOG_LEVEL_MESSAGE, MONO_TRACE_DIAGNOSTICS, msg)
 
 #undef DS_LOG_ALWAYS_1
-#define DS_LOG_ALWAYS_1(msg, data1)
+#define DS_LOG_ALWAYS_1(msg, data1) mono_trace(G_LOG_LEVEL_MESSAGE, MONO_TRACE_DIAGNOSTICS, msg, data1)
 
 #undef DS_LOG_ALWAYS_2
-#define DS_LOG_ALWAYS_2(msg, data1, data2)
+#define DS_LOG_ALWAYS_2(msg, data1, data2) mono_trace(G_LOG_LEVEL_MESSAGE, MONO_TRACE_DIAGNOSTICS, msg, data1, data2)
 
 #undef DS_LOG_INFO_0
-#define DS_LOG_INFO_0(msg)
+#define DS_LOG_INFO_0(msg) mono_trace(G_LOG_LEVEL_INFO, MONO_TRACE_DIAGNOSTICS, msg)
```

Comment:
If you run on info without enabling diagnostic server reverse connect you will get one additional logging:

Mono: ipc_stream_factory_build_and_add_port - Ignoring default LISTEN port

Reply:
And when you use a reverse connect port, this will be logged at info:

Mono: ds_ipc_stream_factory_configure - Attempted to create Diagnostic Port from "127.0.0.1:9000,nosuspend".
Mono: ds_ipc_stream_factory_configure - Diagnostic Port creation succeeded
Mono: ipc_stream_factory_build_and_add_port - Ignoring default LISTEN port

even if the runtime fails to connect towards end point (logged at debug level) since that will be done in loop with 500 ms timeout.